### PR TITLE
Analyst1 - Rebrand illuminate to Analyst1

### PR DESCRIPTION
**Pull request by runt9**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Recently AnalystPlatform has rebranded itself as Analyst1, consolidating both the company name, AnalystPlatform, and our flagship product, illuminate, under the same name. This Pull Request is opened to update our integration to reflect the appropriate rebranding.

One note is it appears there is also some test data for the CortexXDR integration and I wasn't sure if it was safe to update said test data to reflect the rebranding or not.

### CODE REVIEW

**Comment by runt9** [comment]
Looks like I'm having issues with the release notes being accepted. I renamed the Packs directory from `illuminate` to `Analyst1` and I think that's throwing off the `1.0.1` and `1.0.2` release notes, and the validator is treating this update as a "new" pack and thus is saying I should remove the `1.0.3` release notes. What's the right way to proceed?

**Comment by barchen1** [comment]
> Looks like I'm having issues with the release notes being accepted. I renamed the Packs directory from `illuminate` to `Analyst1` and I think that's throwing off the `1.0.1` and `1.0.2` release notes, and the validator is treating this update as a "new" pack and thus is saying I should remove the `1.0.3` release notes. What's the right way to proceed?

@runt9 You should threat this pack as a new pack and delete the ReleaseNotes folder.

**Comment by runt9** [comment]
> @runt9 You should threat this pack as a new pack and delete the ReleaseNotes folder.

@barchen1 Thank you, I'll be sure to do that! In addition, I modified the illuminate -> Analyst1 name in other Packs. Should I generate release notes for those Packs, or should I not change them and allow the developers of those Packs to update them?

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Here are some notes about this PR:
The changes you made here break BC for users who are already use this integration. 
Here is my suggestion:
1. Change the pack directory to its original name- illuminate.
    - changing the folder name will create a new pack and the customers will not be notified on this update.
2. The `name` field in the pack_metadata file is the name of the pack that the customers see so you can leave it as Analyst1.
3. Create a duplicate integration of illuminate which will be named as `Analyst1` and deprecate the `illuminate` integration. 
for deprecating an integration you need to:
     - add `deprecated: true` filed to illuminate yml file and 
     -  add `tests:
                  - No Tests - Deprected` to the illuminate yml file.
     - add `hidden: true` filed to illuminate playbooks. 

Update me what you would like to do - break BC or do the changes as I suggested.
Either way you choose the pack folder needs to be named as illuminate.
Let me know if you have any questions.

**Comment by runt9** [comment]
@barchen1 Thank you for your review. I just merged the upstream branch in and did a hard reset so I can apply the changes on a fairly clean branch without hurting the PR. I will have the proper commit in shortly.

**Comment by barchen1** [comment]
@runt9 From your last commit I understand you choose to deprecate illuminate.
If you want to deprecate this integration you need to have only one pack name `illuminate`
Under this pack you will have 2 integrations - `illuminate` and `Analyst1`.
You don't need to add a pack named `Analyst1`
Please reach me out on Slack if you need help with that.

File: Packs/illuminate/Integrations/illuminate/illuminate.yml [hunk 1]
```diff
@@ -2,9 +2,16 @@ commonfields:
...
 category: Data Enrichment & Threat Intelligence
-description: This integration utilizes AnalystPlatform's illuminate system to enrich Demisto indicators with data provided by the illuminate REST API, such as actor and malware information, activity and reported dates, evidence and hit counts, and more.
+description: Deprecated. This integration utilizes AnalystPlatform's illuminate system to enrich
+  Demisto indicators with data provided by the illuminate REST API, such as actor
+  and malware information, activity and reported dates, evidence and hit counts, and
+  more.
+deprecated: true
+tests:
+  - No Tests
+  - Deprecated
```
Comment (barchen1) [inline Packs/illuminate/Integrations/illuminate/illuminate.yml:14]:
```suggestion
tests:
  - No Tests  - Deprecated
```

File: Packs/Analyst1/Integrations/Analyst1/Analyst1.yml [hunk 2]
```diff
@@ -0,0 +1,609 @@
...
+      description: The AlienVault OTX vendor.
+      type: String
+    description: Queries the Analyst1 REST API and enriches the given URL with Analyst1
+      indicator data.
+  dockerimage: demisto/python3:3.8.5.11789
+  runonce: false
+  subtype: python3
+fromversion: 5.0.0
+tests:
+- No tests (auto formatted)
```
Comment (barchen1) [inline Packs/Analyst1/Integrations/Analyst1/Analyst1.yml:609]:
```suggestion
- Analyst1 Integration Demonstration - Test
```

**Review by barchen1 — CHANGES_REQUESTED** [review]

**Review by barchen1 — APPROVED** [review]

**Pull request merged**